curriculum/js1/6.js

```diff
@@ -5,8 +5,15 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, possibleDivisor = a -1, sum = 0) => {
+  if (possibleDivisor === 1) {
+    return sum;
+  }
+  if (a % possibleDivisor === 0) {
+    sum = sum + possibleDivisor;
+  }
```

Comment:
This is good. If you're looking for ways to improve/shorten the code, consider:
* condensing everything onto one line
* use the [addition assignment operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Assignment_Operators#Addition_assignment)
```javascript
if (a % possibleDivisor === 0) sum += possibleDivisor;
```